src/styles/globals.css

```diff
@@ -63,14 +63,4 @@ input[type='number'] {
   fill: var(--color-logo-background);
 }
 
-.illustration-main-stroke {
-  stroke: var(--color-primary-main);
-}
-
-.illustration-light-stroke {
-  stroke: var(--color-border-main);
-}
-
-.illustration-background-stroke {
-  stroke: var(--color-logo-background);
-}
+/* It's not possible to set `stroke` via a class as it overrides other properties */
```

Comment:
What do you mean? Which other properties does it override?

Reply:
By setting `stroke` via a class, it overrides all other associated stroke properties, i.e. `stroke-width`, `stroke-linecap` etc.

Reply:
It seems like we've found a way. `stroke` needs to exists _as well as_ the class. I'll revert my change above.